### ISSUE TITLE
MuseSounds Suddenly Unavailable After MuseHub Update and File Migration

### ISSUE THREAD

**Issue by mpupster** [musescore/MuseScore#25722]
### Issue type

VST bug

### Description with steps to reproduce

1. Start with functioning MuseScore 4.3.X, MuseHub V1, and MuseSounds (mainly brass and woodwinds) installed.
2. Update MuseHub and MuseScore when prompted by old MuseHub
    - Screen appears saying file migration is taking place, click 'OK'
3. Proceed with using program for the rest of the day, no problems.
4. Log in next day, and see that all parts in mixer have reverted to MSBasic soundfont.
5. Attempt to remove and reinstall MuseSounds.
    - Does not work
6. Attempt to remove and reinstall MuseScore and MuseHub.
    - Does not work
7. Attempt to change soundfont folder within MuseScore program (to match MuseHub's default MuseSound installation folder)
    - Does not work
8. Attempt vice versa
    - Does not work

### Supporting files, videos and screenshots

![Screenshot 2024-12-03 175840](https://github.com/user-attachments/assets/5ec7750b-8856-43de-9649-160cb3daa4f7)
![Screenshot 2024-12-03 175758](https://github.com/user-attachments/assets/f65510e7-2185-42c9-ad00-0c4bfd2b290d)
![Screenshot 2024-12-03 175731](https://github.com/user-attachments/assets/518b758e-3e32-40d8-a70b-3743abeb8c4a)
![Screenshot 2024-12-03 175645](https://github.com/user-attachments/assets/c771a65d-d2b2-40a9-a8f9-e7363896ee50)
![Screenshot 2024-12-03 175628](https://github.com/user-attachments/assets/5271de90-1b53-43f8-afbc-74ceb2cca579)
![Screenshot 2024-12-03 175615](https://github.com/user-attachments/assets/3f9dc6ed-08a5-46c2-88e7-fc54ceff65e8)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.3-242971445, revision: github-musescore-musescore-eb6b367

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11

### Additional context

Apologies for any formatting mistakes or missing information. I am not a tech savvy individual. I came here because I saw a lot of similar issues online but none of what solved their problems worked for me. At the end of my rope. I want my MuseSounds back.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by DominicMaas** [comment]
Can you confirm that the following file exists on your device? 

`%localappdata%\MuseSampler\lib\MuseSamplerCoreLib.dll`

If so, we can try brute forcing this. 

1. Close MuseScore Studio and MuseHub (make sure MuseHub if fully closed by checking the system tray)
2. Delete `%localappdata%\Muse Hub\.receipt`
3. Open MuseHub and wait for a few minutes (1-5)
4. Launch MuseScore Studio and check if the sounds are present.

**Comment by mpupster** [comment]
There is no such file in that folder, only protLib.dll

**Comment by mpupster** [comment]
I found the file you mentioned, however these steps did not resolve the issue.

**Comment by mpupster** [comment]
Solved!

Extremely unique issue; My date & time settings.

I had just moved to the Pacific region when I updated my programs, and for whatever reason, my time zone was not updating automatically, so I manually changed the time in settings and forgot about it.

I dove a little deeper into it today due to other odd effects that was having (e.g. newly sent discord messages saying "sent in 8 hours"). Upon fixing the root issue, I saw a message from MuseHub saying something to the effect of "time synchronization failed" and it clicked in my head.

One last uninstall the reinstall of the program and sounds did the trick!
